templates/map.html

```diff
@@ -133,7 +133,7 @@
 {% if show_featured %}
   <div class="section">
     {% if future_featured %}
-      <h2>This map will be a featured map.</h2>
+      <h2>This map will be a featured map on {{map.featured_date|date:"Y-m-d"}}</h2>
```

Comment:
"This map will be featured on {{date}}." maybe?  Wanna say the "was featured" wording is similar, so if you _do_ decide to change this one...